ddtrace/tracer/log.go

```diff
@@ -0,0 +1,99 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-2020 Datadog, Inc.
+
+package tracer
+
+import (
+	"encoding/json"
+	"fmt"
+	"math"
+	"net/http"
+	"runtime"
+	"time"
+
+	"gopkg.in/DataDog/dd-trace-go.v1/internal/globalconfig"
+	"gopkg.in/DataDog/dd-trace-go.v1/internal/log"
+	"gopkg.in/DataDog/dd-trace-go.v1/internal/version"
+)
+
+const (
+	unknown = "unknown"
+)
+
+type startupInfo struct {
+	Date                  string                 `json:"date"`    // ISO 8601 date and time of start
+	OSName                string                 `json:"os_name"` // Windows, Darwin, Debian, etc.
+	OSVersion             string                 `json:"os_version"`
+	Version               string                 `json:"version"`              // Tracer version
+	Lang                  string                 `json:"lang"`                 // "Go"
+	LangVersion           string                 `json:"lang_version"`         // Go version, e.g. go1.13
+	Env                   string                 `json:"env"`                  // Tracer env
+	Service               string                 `json:"service"`              // Tracer Service
+	AgentURL              string                 `json:"agent_url"`            // The address of the agent
+	AgentError            error                  `json:"agent_error"`          // Any error that occurred trying to connect to agent
+	Debug                 bool                   `json:"debug"`                // Whether debug mode is enabled
+	AnalyticsEnabled      bool                   `json:"analytics_enabled"`    // True if there is a global analytics rate set
+	SampleRate            float64                `json:"sample_rate"`          // The default sampling rate for the priority sampler
+	SamplingRules         []SamplingRule         `json:"sampling_rules"`       // Rules used by the rules sampler
+	SamplingRulesError    error                  `json:"sampling_rules_error"` // Any errors that occurred while parsing sampling rules
+	Tags                  map[string]interface{} `json:"tags"`                 // Global tags
+	RuntimeMetricsEnabled bool                   `json:"runtime_metrics_enabled"`
+	HealthMetricsEnabled  bool                   `json:"health_metrics_enabled"`
```

Comment:
If you comment all the fields, `gofmt` will automatically align all the comments nicely.